sage/install-tailscale.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+set -e
+
+# Set what version of tailscale you would like to install here.
+# You can find the latest version at https://pkgs.tailscale.com/stable/#static
+export TAILSCALE_VERSION=1.90.2
+
+echo
+echo "Installing tailscale ${TAILSCALE_VERSION} for Kobo Sage!"
+uname -a
+echo
+
+echo "Installing iptables into /sbin and /lib ..."
+cp binaries/iptables/sbin/* /sbin
+cp binaries/iptables/lib/* /lib
```

Comment:
```suggestion
cp -R binaries/iptables/lib/* /lib
```

The `xtables` directory would otherwise cause cp to exit 1 and abort the script.